runners/flink/pom.xml

```diff
@@ -111,48 +107,6 @@
 
     </pluginManagement>
 
-
-    <plugins>
-      <!-- Java compiler -->
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.1</version>
-        <configuration>
-          <source>1.7</source>
-          <target>1.7</target>
-        </configuration>
-      </plugin>
-
-      <!-- Maven minimum version check -->
```

Comment:
+1 Makes sense to enforce a minimum Maven version, especially with regard to plugin behaviors varying from version to version.